openxr/examples/fullscreen.vert

```diff
@@ -1,8 +1,27 @@
 #version 450
+#extension GL_EXT_multiview : require
+
+layout(location = 0) in vec3 vPosition;
 
 layout(location = 0) out vec2 screen_coords;
+layout(location = 1) out vec3 vColor;
+
+layout(push_constant) uniform ModelTransform {
+    mat4 model;
+};
+
+struct Transform {
+    mat4 view;
+    mat4 proj;
+};
+
+layout(binding = 0) uniform Transforms {
+    Transform transform[2];
+};
 
 void main()  {
-    screen_coords = vec2((gl_VertexIndex << 1) & 2, gl_VertexIndex & 2);
-    gl_Position = vec4(screen_coords * 2.0f + -1.0f, 0.0, 1.0f);
+    vec4 world_pos = model * vec4(vPosition, 1.0);
+    vec4 pos = transform[gl_ViewIndex].proj * transform[gl_ViewIndex].view * world_pos;
+    gl_Position = pos;
+    vColor = vec3(gl_VertexIndex/10.0, world_pos.gb);
```

Comment:
If this is no longer emitting a fullscreen triangle, the file should be renamed.

Reply:
Renamed to transform.vert